polyfills/Element/prototype/matches/config.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"browsers": {
 		"ie": "6 - *",
+		"ie_mob": "10 - *",
```

Comment:
@CheloXL Would you be able to post a screenshot of the tests passing on IE Mobile 10 & 11?